test/Groups/matrixgroups.jl

```diff
@@ -313,7 +313,7 @@ end
    end
    @test N==99
 
-   @test Set(elements(G))==Set([x for x in G])
+   @test Set(collect(G))==Set([x for x in G])
```

Comment:
I am not sure that this tests anything meaningful anymore. Perhaps this would make more sense:
```suggestion
   @test length(Set(G)) == order(G)
```